src/smart-components/access-management/users-and-user-groups/users/UsersTable.tsx

```diff
@@ -210,6 +211,23 @@ const UsersTable: React.FunctionComponent<UsersTableProps> = ({ onAddUserClick,
       id: user.username,
       is_active: user.is_active,
       row: [
+        [
+          <OrgAdminDropdown
```

Comment:
We should hide this behind a feature flag `platform.rbac.common-auth-model` so if it's not available to show the org.admin as previously. Also only org. admins can change this, so we should toggle between this and static field based on rights.

Reply:
Fixed